platforms/documentation/docs/src/docs/userguide/authoring-builds/other/working_with_files.adoc

```diff
@@ -930,3 +950,20 @@ include::sample[dir="snippets/files/archives/groovy",files="build.gradle[tags=re
 
 Often you will want to publish an archive, so that it is usable from another project.
 This process is described in <<cross_project_publications.adoc#cross_project_publications,Cross-Project publications>>.
+
+[[sec:symlinks_archives]]
+=== Symbolic links in archives
+
+Archives tasks support <<#sec:symlinks,`linksStrategy` property>>.
+
+Note: if any source of a spec is an archive ([`zipTree`](javadoc/org/gradle/api/file/ArchiveOperations.html#zipTree-java.lang.Object-) or [`tarTree`](javadoc/org/gradle/api/file/ArchiveOperations.html#tarTree-java.lang.Object-), to be more specific), then the default value is `LinksStrategy.PRESERVE_RELATIVE`.
+
+Since an archive is treated as a self-contained file system, there is limited support for some scenarios.
+To be specific, no link would be followed if it points outside the archive.
+To mitigate that, either exclude them or use `LinksStrategy.PRESERVE_ALL` and another copy task to achieve the desired result.
+
+Another caveat for symlinks handling is related to tar archives.
+Since they are essentially a stream of data and do not support random access, it is not possible to follow a symlink to a file that is not yet processed.
```

Comment:
```suggestion
Since they are essentially a data stream and do not support random access, it is impossible to follow a symlink to a file that still needs to be processed.
```